packages/dts-generator/src/types/ast.d.ts

```diff
@@ -196,15 +196,16 @@ export interface ExperimentalDesc extends Description {
 
 // Types
 
-export type Type =
+export type Type = (
   | TypeReference
   | TypeLiteral
   | UnionType
   | IntersectionType
   | ArrayType
   | FunctionType
   | LiteralType
-  | NativeTSTypeExpression;
+  | NativeTSTypeExpression
+) & { repeatable?: boolean };
```

Comment:
It is an anachronism in JSDoc that the "repeatable" information is attached to the type (because JSDoc has no parameter names in function signatures, only types) and our api.json inherited this from JSDoc.

I would rather convert this in the ts-ast-type-builder as we do already for the [optional flag](https://github.com/SAP/ui5-typescript/blob/main/packages/dts-generator/src/utils/ts-ast-type-builder.ts#L117).